src/Traits/Categories/AttachmentHelpers.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace NotificationChannels\OneSignal\Traits\Categories;
+
+trait AttachmentHelpers
+{
+    /**
+     * Set an Image only for iOS.
+     *
+     * @param string $imageUrl
+     *
+     * @return $this
+     */
+    public function setIosAttachment(string $imageUrl)
+    {
+        return $this->setParameter('ios_attachments', ['id1' => $imageUrl]);
```

Comment:
Perhaps this should cover single and multiple values - just like subject and body with a pre-check

```php
    /**
     * Set an Image only for iOS.
     *
     * @param string|array $imageUrl
     *
     * @return $this
     */
    public function setIosAttachment($imageUrl)
    {
        $imageUrl = is_array($imageUrl) ? $imageUrl : ['id1' => $imageUrl];
        
        return $this->setParameter('ios_attachments', $imageUrl);
    }
```

Then you could ditch the `setIosAttachments()` method

Reply:
i'm okay with this, but i the other methods accepts only one string param, so i think people would expect that this method makes the same. If they want to add more than one attachment they could think that this is possible on the other methods too...